binding/binding.go

```diff
@@ -0,0 +1,27 @@
+package binding
+
+import "sync"
+
+type UnitBinding interface {
+	addListener(func(interface{}))
+	Set(interface{})
+}
+
+type BaseBinding struct {
```

Comment:
I think we could keep this un-exported - unless we want to support third party bind types? (I think that shouldn't be needed)